Marlin/Conditionals_LCD.h

```diff
@@ -108,7 +108,28 @@
     #define ULTIPANEL
     #define NEWPANEL
 
+  #elif ENABLED(CR10_STOCKDISPLAY)
+  
+  #define REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
+  #ifndef ST7920_DELAY_1
+    #define ST7920_DELAY_1 DELAY_2_NOP
   #endif
+  #ifndef ST7920_DELAY_2
+    #define ST7920_DELAY_2 DELAY_2_NOP
+  #endif
+  #ifndef ST7920_DELAY_3
+    #define ST7920_DELAY_3 DELAY_2_NOP
+  #endif
+  #elif ENABLED (MKS_12864OLED) 
+  
+    #define U8GLIB_SH1106 
+    #define REPRAP_DISCOUNT_SMART_CONTROLLER 
+    #define NEWPANEL 
+
+  #elif ENABLED(MKS_MINI_12864)
```

Comment:
Historically, users reverse the connectors.  This is not a problem isolated to MKS_MINI12864.  See my comments in the discussion regarding this.

Reply:
I am a EE, it is incorrect to install them backwards, defeats the reason for the key.  I believe it is a trick by the vendor to keep everything single sourced.

Reply:
comments removed